docs/api-reference/dynamic-configuration-api.md

```diff
@@ -3,73 +3,663 @@ id: dynamic-configuration-api
+
+* `count`
+  * Type: Integer
+  * Limit the number of results to the last `n` entries.
+
+#### Responses
+
+<Tabs>
+
+<TabItem value="16" label="200 SUCCESS">
+
+
+*Successfully retrieved history*
+
+</TabItem>
+</Tabs>
+
+---
+
+#### Sample request
+
+The following example retrieves the dynamic configuration history between `2022-07-13` and `2024-07-19`. The response is limited to 10 entries.
+
+<Tabs>
+
+<TabItem value="17" label="cURL">
+
+
+```shell
+curl "http://ROUTER_IP:ROUTER_PORT/druid/indexer/v1/worker/history?interval=2022-07-13%2F2024-07-19&count=10"
+```
+
+</TabItem>
+<TabItem value="18" label="HTTP">
+
+
+```HTTP
+GET /druid/indexer/v1/worker/history?interval=2022-07-13%2F2024-07-19&count=10 HTTP/1.1
+Host: http://ROUTER_IP:ROUTER_PORT
+```
+
+</TabItem>
+</Tabs>
+
+#### Sample response
+
+<details>
+  <summary>Click to show sample response</summary>
+
+```json
+[
+    {
+        "key": "worker.config",
+        "type": "worker.config",
+        "auditInfo": {
+            "author": "",
+            "comment": "",
+            "ip": "127.0.0.1"
+        },
+        "payload": "{\"type\":\"default\",\"selectStrategy\":{\"type\":\"fillCapacityWithCategorySpec\",\"workerCategorySpec\":{\"categoryMap\":{},\"strong\":true}},\"autoScaler\":null}",
+        "auditTime": "2023-10-03T21:49:49.991Z"
+    }
+]
+```
+
+</details>
+
+### Get an array of worker nodes in the cluster
+
+Returns an array of all the worker nodes in the cluster along with its corresponding metadata.
+
+<code class="getAPI">GET</code><code>/druid/indexer/v1/workers</code>
+
+#### Responses
+
+<Tabs>
+
+<TabItem value="19" label="200 SUCCESS">
+
+
+*Successfully retrieved worker nodes*
+
+</TabItem>
+</Tabs>
+
+---
+
+#### Sample request
+
+<Tabs>
+
+<TabItem value="20" label="cURL">
+
+
+```shell
+curl "http://ROUTER_IP:ROUTER_PORT/druid/indexer/v1/workers"
+```
+
+</TabItem>
+<TabItem value="21" label="HTTP">
+
+
+```HTTP
+GET /druid/indexer/v1/workers HTTP/1.1
+Host: http://ROUTER_IP:ROUTER_PORT
+```
+
+</TabItem>
+</Tabs>
+
+#### Sample response
+
+<details>
+  <summary>Click to show sample response</summary>
+
+```json
+[
+    {
+        "worker": {
+            "scheme": "http",
+            "host": "localhost:8091",
+            "ip": "198.51.100.0",
+            "capacity": 2,
+            "version": "0",
+            "category": "_default_worker_category"
+        },
+        "currCapacityUsed": 0,
+        "currParallelIndexCapacityUsed": 0,
+        "availabilityGroups": [],
+        "runningTasks": [],
+        "lastCompletedTaskTime": "2023-09-29T19:13:05.505Z",
+        "blacklistedUntil": null
+    }
+]
+```
+
+</details>
+
+### Get scaling events
+
+Returns Overlord scaling events if auto-scaling runners are in use.
```

Comment:
Should auto-scaling have a hyphen or not? See also how it's referred here: https://druid.apache.org/docs/latest/design/overlord/

Reply:
"Autoscaling" appears to be more prevalent in Druid docs.